Fix build and fix dowload kubeconfig (#620)
harvester/docs: docs/rancher/rancher-terraform.md

1. Obtain the kubeconfig for the Harvester Cloud Provider and the Harvester CSI Driver.

    <Tabs>
    <TabItem value="ui" label="UI" default>

>     On the Rancher UI, go to **Virtualization Management** and select
>     **Download Kubeconfig** from the context menu of the corresponding Harvester
>     Cluster.

Reviewer: ```suggestion
    On the Rancher UI, go to **Virtualization Management**. Locate the target Harvester cluster in the list and then select **⋮** > **Download KubeConfig**.
```

Reviewer: For most users, a series of menu choices is easier to understand than UI-specific terms such as "context menu".

Author: Good point. Thanks :smile:
